xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-test/xwiki-platform-office-test-docker/src/test/it/ui/OfficeImporterIT.java

```diff
@@ -17,44 +17,60 @@
  * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
  */
-package org.xwiki.officeimporter.test.ui;
+package ui;
```

Comment:
Not sure if this is a good practice to have such a package name.

Reply:
outch. Indeed good catch I missed this one!